src/components/Book/Books.js

```diff
@@ -1,25 +1,34 @@
 import React from 'react';
-import { useSelector } from 'react-redux';
-import { generate } from 'randomized-string';
-import { alphanumeric } from 'randomized-string/lib/types';
-import Book from './Book';
+import { useSelector, useDispatch } from 'react-redux';
+
+// import PropTypes from 'prop-types';
 import AddBooks from '../Addbooks/AddBooks';
-import './Books.css';
+import { removeBook } from '../../Redux/books/books';
 
 const Books = () => {
-  const book = useSelector((state) => state.booksReducer);
+  const books = useSelector((state) => state.booksReducer);
+  console.log(books);
+  const dispatch = useDispatch();
+
+  const handleClick = (e) => {
+    dispatch(removeBook(e));
+  };
```

Comment:
OPTIONAL

- [ ] The remove book function is not working. You should add it in the last milestone. Please fix it.